cmd/podman/utils.go

```diff
@@ -13,7 +13,7 @@ func getRuntime(c *cli.Context) (*libpod.Runtime, error) {
 	// TODO add option to parse configs (CRI-O configs?) to generate runtimes
```

Comment:
Remove this comment. Perhaps remove all references to CRI-O.  I am fine with doing this as a separate PR>